src/lib/dfp/fill-static-advert-slots.ts

```diff
@@ -21,10 +21,11 @@ const decideAdditionalSizes = (adSlot: HTMLElement): SizeMapping => {
 			desktop: [adSizes.billboard, createAdSize(900, 250)],
 		};
 	} else if (
-		name === 'merchandising-high' &&
-		includeBillboardsInMerchHigh()
+		(name === 'merchandising-high' || name === 'merchandising') &&
+		includeAdsInMerch()
 	) {
 		return {
+			mobile: [adSizes.mpu],
```

Comment:
This will cause all devices mobile and larger to have the MPU size. To exclude MPUs from phablet and above, a new size mapping will need to be created for phablet in `ad-sizes.ts`.

Reply:
I hadn't really ever considered this before, do we do something like this for top-above-nav?

Reply:
For top-above-nav, we define sizes for the tablet viewport in `ad-sizes.ts`. This means that if we do define additional sizes for top-above-nav at the mobile viewport anywhere, they won't be picked up by tablet or above.

Reply:
Oh yes thanks @domlander I can see now how my implementation is totally wrong! I don't have many gaps today, so let me have a quick go at this:

Currently, our `merchandising-high` ad sizes object looks like:

```js
{
  mobile: [
    adSizes.outOfPage,
    adSizes.empty,
    adSizes.merchandisingHigh,
    adSizes.fluid,
  }
}
```

I've updated it dynamically so that it now looks like this:

```js
{
  mobile: [adSizes.mpu],
  desktop: [adSizes.billboard]
}
```

Which is totally wrong 😄 We actually want it to look like this:

```js
{
  mobile: [
    adSizes.outOfPage,
    adSizes.empty,
    adSizes.merchandisingHigh,
    adSizes.fluid,
    adSizes.mpu
  },
  phablet: [
    adSizes.outOfPage,
    adSizes.empty,
    adSizes.merchandisingHigh,
    adSizes.fluid
  }
  desktop: [
    adSizes.outOfPage,
    adSizes.empty,
    adSizes.merchandisingHigh,
    adSizes.fluid,
    adSizes: billboard
  }
}
```

Does that sound about right?

Reply:
That last one looks right to me! Although thinking about it, `phablet` can probably display MPU's. `tablet` is maybe the smallest breakpoint that should not display MPUs.

The above can be achieved by adding the following to `ad-sizes.ts`. I think a comment is necessary to explain why we are duplicating the mapping from mobile to tablet.

```
'merchandising-high': {
		mobile: [
			...
		],
                 // The mobile mapping is given an additional size when filling the slot. We do not want this extra size applied to tablet, so we provide a mapping for tablet here.
                 tablet: [
			adSizes.outOfPage,
			adSizes.empty,
			adSizes.merchandisingHigh,
			adSizes.fluid,
		],
	},
```

I think that later when we remove the AB test, we will be able to specify the sizes in `ad-sizes.ts` directly, instead of having to add the sizes later in `fill-static-advert-slots.ts`.

Reply:
Makes perfect sense, thanks @domlander 🙏